src/library.cpp

```diff
@@ -94,15 +95,24 @@ QStringList Library::listBookIds(const QString &query, const QString &categoryFi
 {
     QStringList list;
     std::vector<std::string> tags;
-    if (categoryFilter != "all") {
+    if (categoryFilter != "all" && categoryFilter != "other") {
         tags.push_back(categoryFilter.toStdString());
     }
+    bool other = false;
+    if (categoryFilter == "other") {
+        other = true;
+        auto categoryList = KiwixApp::instance()->getMainWindow()->getSideContentManager()->getCategoryList();
+        for (auto& category: categoryList) {
+            tags.push_back(category.toLower().toStdString());
```

Comment:
We should not add the tag "other" here. Else we may never be able to have book that have the "other" tag.